README.md

```diff
@@ -75,6 +76,29 @@ Then, use the script:
 ./make_printable.sh
 ```
 
+### Translations 🌍
```

Comment:
**chore:** All other sections start with the Emoji, so maybe we should replace this with "🌍 Translations"?

Reply:
Gosh, true, fixing!